src/components/MissionsItem.js

```diff
@@ -14,8 +20,35 @@ const MissionsItem = (props) => {
         {description}
         <p> </p>
       </td>
-      <td className="memberInfo"><Button type="button" variant="secondary">NOT A MEMBER</Button></td>
-      <td className="joinButton"><Button type="button" variant="outline-secondary">Join Mission</Button></td>
+      {reserved
+        ? (
+          <>
+            <td className="memberInfo activeMember">
+              <Button type="button" variant="info">
+                Active Member
+              </Button>
```

Comment:
For badge react-bootstrap badge component is best, you can replace Button with a badge like, 
 <Badge bg="secondary">Active Member</Badge>